.github/workflows/notify.slack.yml

```diff
@@ -91,7 +98,9 @@ jobs:
       - name: Post to a Failure Message to Slack
         if: ${{ !inputs.is_success }}
         id: slack-failure-message
-        uses: slackapi/slack-github-action@v1.23.0
+        uses: slackapi/slack-github-action@v1.25.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ env.FAILURE_SLACK_WEBHOOK_URL }}
```

Comment:
maybe an env.FAILURE_SLACK_WEBHOOK_URL || env.SLACK_WEBHOOK_URL to maintain existing functionality

Reply:
@tophercf already there on L23 😄 
```
  FAILURE_SLACK_WEBHOOK_URL: ${{ secrets.failure_slack_webhook_url || secrets.SLACK_WEBHOOK_URL }}
```